lib/vanagon/platform/windows.rb

```diff
@@ -55,6 +56,7 @@ def generate_packaging_artifacts(workdir, name, binding)
       def generate_msi_packaging_artifacts(workdir, name, binding)
         FileUtils.mkdir_p(File.join(workdir, "wix"))
         erb_file(File.join(VANAGON_ROOT, "resources/windows/wix/service.component.wxs.erb"), File.join(workdir, "wix", "service.#{name}.wxs"), false, { :binding => binding })
```

Comment:
Not for this PR, but I should have named this file `service.project.wxs.erb` since it contains all the services for the project, not services for individual components.